### PR TITLE
Make sure EV is not polluting our signals

### PR DESCRIPTION
**Pull request by coolo**
adam made me install perl-EV and now I know it's broken. Once it's in,
it will install a SIGCHLD handler reaping all children and we no longer
have any way to get the exit status of isotovideo

### CODE REVIEW

File: script/openqa [hunk 1]
```diff
@@ -27,6 +27,8 @@ $ENV{'MOJO_LISTEN'} ||= 'http://localhost:9526/';
 # allow up to 20GB - hdd images
 $ENV{MOJO_MAX_MESSAGE_SIZE}   = 1024 * 1024 * 1024 * 20;
 $ENV{MOJO_INACTIVITY_TIMEOUT} = 300;
+# the default is EV, and this heavily screws with our children handling
+$ENV{MOJO_REACTOR} = 'Mojo::Reactor::Poll';
```
Comment (aaannz) [inline script/openqa]: Even openQA needs Poll?

Comment (coolo) [inline script/openqa]: I added it in all cases - we used Poll so far anyway. Otherwise we would have noticed ourselves that perl-EV breaks os-autoinst ;)

Comment (coolo) [inline script/openqa]: but openQA has waitpid code too, so better be safe than sorry

Comment (aaannz) [inline script/openqa]: Sure, I was just wondering if you hit something.

**Pull request merged**